sdk/communication/Azure.Communication.Chat/src/Models/CommunicationIdentifierModel.cs

```diff
@@ -6,7 +6,7 @@
 namespace Azure.Communication
 {
     [CodeGenModel("CommunicationIdentifierModel")]
-    internal partial class CommunicationIdentifierModel
+    public partial class CommunicationIdentifierModel
```

Comment:
This type should not be public. The `Serialize` method would convert this `CommunicationIdentifierModel` to `CommunicationIdentifier`

Reply:
ChatMessageReadReceipt has a member called SenderCommunicationIdentifier which is of type CommunicationIdentifierModel and ChatMessageReadReceipt is public.  If I don't set CommunicationIdentifierModel also to public, I would have this error for the SenderCommunicationIdentifier inside ChatMessageReadReceipt after running autorest on the new swagger,:

"Error CS0053 Inconsistent accessibility: property type 'CommunicationIdentifierModel' is less accessible than property 'ChatMessageReadReceipt.SenderCommunicationIdentifier' Azure.Communication.Chat (netstandard2.0) C:\Users\gelli\Desktop\Repos\azure-sdk-for-net\sdk\communication\Azure.Communication.Chat\src\Generated\Models\ChatMessageReadReceipt.cs 38 Active"

Reply:
@glorialimicrosoft we had something similar with other types, lets take a look at this together to figure it out ...
If these internal models get exposed, it will be confusing for the users ..

Reply:
Yeah those properties should become internal and instead a public property of type `CommunicationIdentifier` would be added. Similar to how we convert the Sender property of ChatMessage from `CommunicationIdentifierModel` to `CommunicationIdentifier`

Reply:
The newest commit hid  away the CommunicationIdentifierModel